static/css/restyle/restyle.less

```diff
@@ -840,10 +840,10 @@ input:not(.upvotes):not(.downvotes)[type="submit"],
   box-sizing: border-box;
   color: #fff;
   font-family: @primary-font;
-  font-size: 1.077rem;
+  font-size: 1rem;
```

Comment:
Droping the font-size a touch + decreasing the L+R padding prevents the text wrapping.